.gitignore

```diff
@@ -0,0 +1,4 @@
+/_build
```

Comment:
I'd ignore the `cover` directory too, as it's an artefact generated by tests, and not something that needs to be distributed.